### DISCUSSION TITLE
Politics and source code

### DISCUSSION THREAD

**Discussion by MischaKr** [HerMajestyDrMona/Windows11DragAndDropToTaskbarFix#76] [General]
Looking at the source code I'm seeing a few real life people instead of more traditional variable names. 

It seems a lot safer to substitute all these names for more descriptive variable names so that there are no problems down the line when anyone turns into a monster in the view of the public.

**Comment by HerMajestyDrMona** [comment]
Yes, I agree. There is actually only 1 function where I speeded up and put something "random". 

The story behind it is issue https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix/issues/17.

Someone needed help with remapping keys used by the program (coffee spill on the keyboard). I spent the whole night for update to [ver. 1.8.0.0](https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix/blob/main/CHANGELOG.md#ver-1800----2021-11-02-0700-cet) where [Remapping keys used by the program in order to simulate hotkeys](https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix/blob/main/CONFIGURATION.md#remapping-keys-used-by-the-program-in-order-to-simulate-hotkeys) option was added. I hope this excuses and explains the variable names used.

On the other hand, I'm actually satisfied that it was done this way, because thanks to it a change https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix/commit/f50fdab5bc74891bb6c029f2798634c95726321b could be done. It condemns Putin and generally gives a strong signal what is the program creator's attitude towards criminals like him.
So in short, these variables are unfortunate, but gave me a nice chance to condemn Putin, so I'm happy about it and I don't regret it.

When there's some update of the program, I might (and should) fix some variables as well as finally remove 1/3 of the code where unused functions are left. However, at this point there's nothing more to fix in this program, and Windows 11 build 22621 (most likely 22H2 RTM) adds the native drag-and-drop support. So the only update would probably be something that detects 22621 and displays a message that [Windows11DragAndDropToTaskbarFix](https://github.com/HerMajestyDrMona/Windows11DragAndDropToTaskbarFix) is not longer needed, and asks to remove itself from autostart.
